### ISSUE TITLE
[Question] Is this library maintained and does it use latest upstream yoga?

### ISSUE THREAD

**Issue by MinimogDev** [CodeWitchBella/yoga-wasm#4]
Hey 🙌

I'm currently using `yoga-layout-prebuilt` in my project and was looking to switch to your wasm bindings. After going through the repo everything seems to be ok, I just wanted to ask following questions:

1. Is this lib still maintained - I assume it doesn't need much maintenance anyway? but say in terms of updating dev dependencies etc? And are you open to PR's to update them?

2. Does this need to be updated with upstream yoga code often, or can it be assumed that it always uses latest available code (without the need to make new releases?)

3. I can't for a life of mine get this going with webpack 4/5, any chance you could provide usage example for this library?

**Comment by CodeWitchBella** [comment]
> Hey 🙌
> 
> I'm currently using `yoga-layout-prebuilt` in my project and was looking to switch to your wasm bindings. After going through the repo everything seems to be ok, I just wanted to ask following questions:
> 
> 1. Is this lib still maintained - I assume it doesn't need much maintenance anyway? but say in terms of updating dev dependencies etc? And are you open to PR's to update them?
> 

Not really. It never got used in project long-term (I use react-pdf and was not willing to maintain fork of that to use this). That being said I would still like to use this in the future.

> 2. Does this need to be updated with upstream yoga code often, or can it be assumed that it always uses latest available code (without the need to make new releases?)
> 

This does need to be updated when upstream yoga. That being said I dont actually know what latest version of yoga is... Looking at npm [yoga-layout](https://www.npmjs.com/package/yoga-layout) has latest version 1.9.3 from 2 years ago while yoga-layout-prebuillt has 1.9.6 which seems to be the same as 1.9.3 but with some changes related to the packaging.

[Yoga's github repo](https://github.com/facebook/yoga/releases) shows  1.18.0.

Updating my version consists of two parts: updating yoga itself - this is easy since I did not modify it. I just added info on how that is done in readme.

Second part is updating bindings. Since the bindings are mostly copied from yoga you could just do it again - copy c++ files from yoga, look at the diff and keep changes which are intentional. This is a bit tedious and manual, but should not be that hard.

> 3. I can't for a life of mine get this going with webpack 4/5, any chance you could provide usage example for this library?

If I don't forget I'll create one :-) Do you use yoga-layout-prebuild with some other library so that I can test it?

**Comment by MinimogDev** [comment]
Alright gotcha! If I manage to get this going when you create that webpack example and it all works good, I'd be happy to submit all necessary updates 👍 

I'm working in a private repo atm, but yeh essentially using yoga-layout-prebuilt atm with https://www.pixijs.com there is very very similar project to what I am doing atm available at https://github.com/lunarraid/react-pixi-layout maybe it could help with the test, If not let me know I will try to extract my package from this private monorepo and make a small example one.

**Comment by Skywalker13** [comment]
> Not really. It never got used in project long-term (I use react-pdf and was not willing to maintain fork of that to use this). That being said I would still like to use this in the future.

Hi @CodeWitchBella, do you have the patches somewhere that you've applied to react-pdf in order to use yoga-wasm please?

**Comment by CodeWitchBella** [comment]
> > Not really. It never got used in project long-term (I use react-pdf and was not willing to maintain fork of that to use this). That being said I would still like to use this in the future.
> 
> Hi @CodeWitchBella, do you have the patches somewhere that you've applied to react-pdf in order to use yoga-wasm please?

It's in this pr: https://github.com/diegomura/react-pdf/pull/694

repo: https://github.com/CodeWitchBella/react-pdf/tree/custom-yoga

**Comment by Skywalker13** [comment]
Thank you very much!

**Comment by CodeWitchBella** [comment]
Cross-posting here for anyone following this issue:

#18 
> Since I don't really maintain, or plan to maitain this repo, I figured I'd shout out an alternatives, in case someone lands here from github.
>
> by @shuding: https://github.com/shuding/yoga-wasm-web
